RetailSuiteAccounting/Integration/05_ManageCustomerSupplier.md

```diff
@@ -0,0 +1,97 @@
+[!!Fakturierung](RetailSuiteFaktBase)  
+[!!User Interface debtors/creditors](../UserInterface/02a_DebtorsCreditors.md)  
+
+# Manage the customer/supplier
+
+The *Accounting* module contains a function to create new customers and suppliers, and to enter all relevant information about them. This function is also available in the *Invoicing* module. This information is needed for both modules to operate.
+
+> [Info] Any customer/supplier information created, edited or deleted in the *Accounting* module will be also be created, edited or deleted in the *Invoicing* module, and vice versa. It may be necessary to refresh the page with the F5 key, though, to see the changes in the *Customer/supplier list* tab.
```

Comment:
```suggestion
> [Info] The customer and supplier list in the *Accounting* and in the *Invoicing* module are synchronized with each other. Therefore, it may be necessary to refresh the list to display any changes that have been made.
```